devel/api.md

```diff
@@ -656,6 +663,46 @@ will also change the interfaces that are visible in the Nmstate state. Reducing
 the total VFs without setting `Interface.STATE` to `InterfaceState.ABSENT` for
 the corresponding interfaces might lead to an undefined system state or error.
 
+### `Ethernet.SRIOV.VFS_SUBTREE`
+
+If `Ethernet.SRIOV.TOTAL_VFS` is 1 or greater, the `Ethernet.SRIOV.VFS_SUBTREE`
+dictionary list allows to configure the VFS special options. Nmstate supports the
+following VF options:
+
+#### `Ethernet.SRIOV.VFS.ID`
+
+Type: `integer`
+
+ID of the VF to configure.
+
+#### `Ethernet.SRIOV.VFS.MAC_ADDRESS`
+
+Type: `string` in the format of `EA:60:E4:08:17:F1`. Case insensitive.
```

Comment:
Where does this MAC address come from? How about using something artificial like `11:22:33:44:55:66` to make clear it is an example? Maybe there are even special example MAC addresses.

Reply:
What is the default? Also wondering for the other options.

Reply:
I have trying to get more information. It seems the defaults depends on the HW. So.. I think we should specify a default for these values.